azure-pipelines.yml

```diff
@@ -238,16 +228,9 @@ stages:
             artifact: Maestro.ScenarioTests
             displayName: Publish Maestro Scenario Tests
 
-          # Reenable tagging when stable package versions can be generated.
-          # https://github.com/dotnet/arcade-services/issues/59
-          # - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.SourceBranchName'], 'production')) }}:
-            # - powershell: eng\create-tag.ps1 -AccessToken $(System.AccessToken)
-
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
   - template: eng\common\templates\post-build\post-build.yml
     parameters:
-      # Symbol validation isn't being very reliable lately. This should be enabled back
-      # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
```

Comment:
This is removed but the symbol validation below stays turned off. Should we not keep it or flip the boolean?

Reply:
We can try flipping it on, I'll run these changes on the internal pipeline, see if they behave as expected

Reply:
https://dev.azure.com/dnceng/internal/_build/results?buildId=2194790&view=results

Reply:
Actually, I'll add the missing tests in a separate PR, think that will be better